CMakeLists.txt

```diff
@@ -629,6 +630,11 @@ target_link_libraries(
     ${EXTRA_LIBRARIES}
 )
 
+# backtrace functions are not part of libc like on Linux platform
+if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
```

Comment:
Do you know if it would be needed on OpenBSD too?

Reply:
OpenBSD does not have execinfo implementation in libc too, so yes. But FreeBSD have libexecinfo included in base system while OpenBSD need to install it as a port. This means
OpenBSD would need check for libexecinfo, and probably setting includes for build (execinfo.h in /usr/include vs in /usr/local/include). I'm starting to speculating here and don't want add to the patch anything I'm not sure 100%.